ocm_python_wrapper/cluster.py

```diff
@@ -388,3 +396,24 @@ def _wait_for_rhmi_resource():
         ResourceEditor(
             patches={rhmi: {"spec": {"useClusterStorage": "false"}}}
         ).update()
+
+    @staticmethod
+    def create_rhods_brew_config():
+        if not os.getenv("PS_DATA"):
+            raise KeyError("Pull-secret data must be specified")
```

Comment:
`Pull-secret data must be specified")` specified where?

Reply:
Updated error message with env var name to pass the pull-secret data